### PR TITLE
Loki: Remove any from public/app/plugins/datasource/loki/components/LokiOptionFields.test.tsx

### LINKED ISSUE #53133: Loki: Remove `any` from public/app/plugins/datasource/loki/components/LokiOptionFields.test.tsx
In `public/app/plugins/datasource/loki/components/LokiOptionFields.test.tsx` we have couple of `any` types or `as any` type assertions. The goal is to remove all of them from this test file.

### PR DESCRIPTION
**Pull request by gitstart**
**What this PR does / why we need it**:
Removes `any` from `LokiOptionFields.test.tsx`

**Which issue(s) this PR fixes**:

Fixes #53133

**Special notes for your reviewer**:

### CODE REVIEW

File: public/app/plugins/datasource/loki/components/LokiOptionFields.test.tsx
Comment (matyax) [inline public/app/plugins/datasource/loki/components/LokiOptionFields.test.tsx:14]: This `propOverrides` is not used at all in this test, so it would be preferable to remove it.

File: public/app/plugins/datasource/loki/components/LokiOptionFields.test.tsx
Comment (matyax) [inline public/app/plugins/datasource/loki/components/LokiOptionFields.test.tsx:26]: Some of the props here, that are later spreaded to the component, don't exist in `LokiOptionFieldsProps`.

**Review by matyax — CHANGES_REQUESTED** [review]
Thank you for your contribution! I would suggest to remove `propOverrides` from `setup()`, and ensure that only attributes from `LokiOptionFieldsProps` are passed to the rendered component.

**Comment by matyax** [comment]
Thank you for reiterating on this PR!

**Comment by gitstart** [comment]
Hey @matyax I think this needs your approval before it can be merged

**Review by matyax — APPROVED** [review]

**Comment by matyax** [comment]
Thanks for the reminder @gitstart

**Comment by gitstart** [comment]
Thank you for approving the PR @matyax but this is still not merged. How can we get this merged? @matyax

**Comment by gitstart** [comment]
> @gitstart I'm waiting for CI. Would you kindly be a little bit patient please?

Will do, sorry about this @matyax

**Comment by matyax** [comment]
There's a check that's failing to run. This PR will be merged once that goes back to normal.

**Comment by matyax** [comment]
@gitstart Could you please send an empty commit (`git commit --amend`) to re-trigger the failing job? Thanks.

**Comment by gitstart** [comment]
@matyax PR has been updated. Please take a look 🙏

**Review by matyax — APPROVED** [review]

**Comment by matyax** [comment]
Thank you @gitstart !

**Pull request merged**
